src/ECR.jl

```diff
@@ -1,11 +1,15 @@
 module ECR
 
-using AWSCore
-using AWSCore.Services: ecr
+using AWS
 using Base64
 using Mocking
 
-get_authorization_token(config; kwargs...) = ecr(config, "GetAuthorizationToken"; kwargs...)
+# Improper casing to avoid issues with Module name and AWS.AWSService
+@service Ecr
+
+function get_authorization_token(config::AWSConfig, params::AbstractDict)
```

Comment:
```suggestion
function get_authorization_token(config::AWSConfig, params::AbstractDict=Dict{String, Any}())
```
Then you don't have to add the empty dict argument below

Reply:
I had tried this but ran into a lot of problems w/ the CI and Mocking. I'll try again.

Reply:
Specifically around dropping [this](https://github.com/JuliaCloud/AWSTools.jl/pull/27/files#diff-e904f31156be89addefae4f45093c1d136fdc987af091045b7bf102558c10554R33).

Reply:
Okay latest commit fixes this in a better way.